daemon/graphdriver/quota/projectquota.go

```diff
@@ -1,4 +1,4 @@
-// +build linux,!exclude_disk_quota
+// +build linux,!exclude_disk_quota linux,cgo
```

Comment:
The intent here is `linux AND !exclude_disk_quota AND cgo`, right?  So shouldn't this be changed to:

```suggestion
// +build linux,!exclude_disk_quota,cgo
```

Reply:
Shoudn't it be when "exclude_disk_quota" OR "linux/cgo" ?

Reply:
I don't think so, because then if `cgo` is supported, it will be built no matter what, even if `exclude_disk_quota` is set.

Reply:
True! This is confusing :)

Reply:
I don't think so since it would be built if `linux,exclude_disk_quota` OR `linux,!cgo`... right?

Reply:
The way this is currently written (`linux,!exclude_disk_quota linux,cgo`), it translates to `(linux AND !exclude_disk_quota) OR (linux AND cgo)`.

Reply:
Doh sorry, missed your first comment (it didn't hit my email yet) -- no, I think `projectquota_unsupported.go` is correct because it needs to be `linux AND (!cgo OR exclude_disk_quota)`, which would be `linux,exclude_disk_quota linux,!cgo`.

Reply:
Nice... I've pushed the update.
Thanks! 💯